vendor/github.com/sirupsen/logrus/go.mod

```diff
@@ -8,3 +8,5 @@ require (
 	github.com/stretchr/testify v1.2.2
 	golang.org/x/sys v0.0.0-20190422165155-953cdadca894
 )
+
+go 1.13
```

Comment:
I've been seeing this line get added automatically all the time when working locally. It'll be nice to have it committed.